internals/overlord/checkstate/manager.go

```diff
@@ -419,6 +428,8 @@ type checkData struct {
 	changeID     string
 	successCount int64
 	failureCount int64
+	refresh      chan refreshInfo
+	result       chan error
```

Comment:
What I intended here is that this `result` field would be completely removed from here, and only be in the `refreshInfo` struct, and a new channel created there each time a refresh is requested (because the result "belongs" to the refresh).